src/content/ccip/api-reference/ccip-receiver.mdx

```diff
@@ -5,14 +5,27 @@ title: "CCIPReceiver API Reference"
 ---
 
 import { Aside, CopyText } from "@components"
+import { Tabs } from "@components/Tabs"
 
 <Aside type="note" title="Add Chainlink CCIP to your project">
```

Comment:
At this point I'm sure we should just take this part out of the `Aside` entirely, but I thought it was mildly interesting that the `Tabs` component works inside the `Aside` component so I wanted to share.